lib/spark_api/models/newsfeed.rb

```diff
@@ -0,0 +1,15 @@
+module SparkApi
+  module Models
+    class Newsfeed < Base 
+      
+      extend Finders
+
+      self.element_name = 'newsfeeds'
+
+      def update!(arguments={})
+        connection.put( "/newsfeeds/#{self.Id}", arguments ).first
```

Comment:
One issue here: the instance attributes will get out of sync with what's actually persisted to the API. If you pull in the `Savable` concern, that'll provide the `update!` method as well as handle the dirty tracking. The only difference is the usage would look like:

```ruby
newsfeed = Newsfeed.find('1234')
newsfeed.Active = false
newsfeed.save!
```